HousingSearchListener/V1/UseCase/AddPersonToTenureUseCase.cs

```diff
@@ -112,7 +112,7 @@ private static List<HouseholdMembers> GetHouseholdMembersFromEventData(object da
 
         private static T ConvertFromObject<T>(object obj) where T : class
         {
-            return JsonSerializer.Deserialize<T>(JsonSerializer.Serialize(obj), JsonOptions.CreateJsonOptions());
+            return JsonSerializer.Deserialize<T>(obj.ToString(), JsonOptions.CreateJsonOptions());
```

Comment:
What is the reason behind changing this?

Reply:
There was issue in other palce, already rolled back this change